src/coreclr/jit/codegenarmarch.cpp

```diff
@@ -2307,8 +2313,8 @@ void CodeGen::genCallInstruction(GenTreeCall* call)
 #endif // TARGET_ARM
             }
         }
-#if FEATURE_ARG_SPLIT
-        else if (curArgTabEntry->IsSplit())
+#if FEATURE_ARG_SPLIT_SUPPORTED
```

Comment:
Some minor comments about `FEATURE_ARG_SPLIT_SUPPORTED`: it's not clear the renaming of the ifdef matters. Also, now that it is always enabled for arm32 & arm64, we could even remove the ifdefs in arm32/arm64-only code files, like codegenarmarch.cpp. Also, here there's no need to check for `compFeatureArgSplit()` since `IsSplit()` is also doing that.